### PR TITLE
Update docs with Kubic project plans

### PR DESCRIPTION
**Pull request by lsm5**
Background info:
Support for older distros:
https://podman.io/blogs/2021/03/02/podman-support-for-older-distros.html

Support for current and upcoming distros:
https://hackmd.io/fc1zraYdS0-klJ2KJcfC7w?view#Podman-Packages-on-Kubic

Also remove @baude's COPR repo link as it may not be maintained going
forward.

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@TomSweeneyRedHat @rhatdan @vrothberg PTAL

### CODE REVIEW

File: getting-started/installation.md [hunk 1]
```diff
@@ -269,37 +261,21 @@ sudo apt-get -y upgrade
...
-sudo yum -y copr enable lsm5/container-selinux
-sudo yum -y install podman
-```
-
 #### [CentOS](https://www.centos.org)
 
-Podman is available in the default Extras repos for CentOS 7 and in
-the AppStream repo for CentOS 8 and Stream, however the available version often
-lags the upstream release.
+Podman is available in the AppStream repo for CentOS 8 and Stream and are also the preferable choice
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md]:
```suggestion
Podman is available in the AppStream repo for CentOS 8 and Stream, and these are also the preferred choice for use in production environments. However, the available version may be older than the upstream release.


```

File: getting-started/installation.md [hunk 2]
```diff
@@ -98,6 +102,14 @@ sudo dnf -y install podman
 sudo dnf -y update
 ```
 
+NOTE:
+1. CentOS 7 Kubic repo will receive no further updates after Podman v3.0.
+2. CentOS 8 Kubic repo will continue to exist for the lifetime of CentOS 8 itself.
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md]: I'm a bit unclear, will Podman v3.0+ be supported in CentOS8  Maybe (not sure if this is true/planned):
```suggestion
2. CentOS 8 Kubic repo will continue to receive updates for Podman v3.0+ for the lifetime of CentOS 8 itself.
```

File: getting-started/installation.md (hunk 1 above)
Comment (lsm5) [reply getting-started/installation.md:270]: added

File: getting-started/installation.md [hunk 3]
```diff
@@ -98,6 +102,14 @@ sudo dnf -y install podman
...
+3. Users are recommended to switch to newer versions of CentOS once these deadlines have passed.
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md]: I'm not sure what "deadlines" are.   Maybe:
```suggestion
3. Users are recommended to switch to newer versions of CentOS to receive the most up to date versions of Podman.
```

File: getting-started/installation.md (hunk 2 above)
Comment (lsm5) [reply getting-started/installation.md:107]: added

File: getting-started/installation.md (hunk 3 above)
Comment (lsm5) [reply getting-started/installation.md:108]: added

**Comment by lsm5** [comment]
@TomSweeneyRedHat thanks a lot and keep them coming, I'll add all your suggestions to a single batch and commit them.

**Comment by TomSweeneyRedHat** [comment]
@lsm5 that's all I had, do another push and it should be gtg.

**Comment by lsm5** [comment]
@TomSweeneyRedHat suggestions applied in batch commit.

**Comment by TomSweeneyRedHat** [comment]
LGTM
thanks @lsm5

**Pull request merged**